### DISCUSSION TITLE
For LinearBasicDisplay tracks, display individual transcripts names

### DISCUSSION THREAD

**Discussion by AlexWeinreb** [GMOD/jbrowse-components#3362] [Ideas] [answered]
Let me use Wormbase as an illustrative example. In [JBrowse1](https://wormbase.org/tools/genome/jbrowse-simple/?tracks=Curated_Genes&loc=V%3A10819434..10849804&data=data%2Fc_elegans_PRJNA13758&highlight=), when a gene has several transcripts, in the normal view the transcript names were explicitly annotated on the track:

For example, for the gene _nrx-1_ with its many transcripts:
![image](https://user-images.githubusercontent.com/48498801/204874978-f3514c78-66f4-405c-8336-3667cae20da5.png)

However, on [JBrowse2](https://wormbase.org/tools/genome/jbrowse2/index.html?tracks=c_elegans_PRJNA13758_curated_genes%2Cc_elegans_PRJNA13758_protein_motifs%2Cc_elegans_PRJNA13758_classical_alleles&loc=V%3A10814165..10853210&assembly=c_elegans_PRJNA13758), even the Normal view is quite compact, and displays the gene name but not the transcripts names:

![image](https://user-images.githubusercontent.com/48498801/204875764-ebef00fd-24f2-424a-915c-0e73bc77b01b.png)

As such it becomes hard to identify the transcripts (especially since they are no longer in alphabetical order). Could it be possible to add an "expanded" Display mode that would avoid that problem?

Note: the Wormbase example uses an NCListAdapter, but I see the same thing with a GFF3TabixAdapter track. My feature request applies to all these type of tracks, if I'm not mistaken that is the tracks with a LinearBasicDisplay type.

**Comment by cmdcolin** [comment, marked as answer]
indeed this is not yet available, unfortunately! an issue for tracking is here https://github.com/GMOD/jbrowse-components/issues/241

**Reply by AlexWeinreb** [reply]
Sorry I missed the fact it already existed. Thank you!

**Reply by cmdcolin** [reply]
no worries :) it's one of the oldest issues out there by this point, good to get a reminder on it!
